tensorflow/contrib/cmake/tf_tools.cmake

```diff
@@ -147,3 +147,8 @@ target_link_libraries(${benchmark_model} PUBLIC
   ${tf_core_gpu_kernels_lib}
   ${tensorflow_EXTERNAL_LIBRARIES}
 )
+
+install(TARGETS ${proto_text} ${transform_graph} ${summarize_graph} ${compare_graphs} ${benchmark_model}
```

Comment:
I'm not sure to what extent we'd consider these tools part of the public package. The `proto_text` tool is certainly for internal build use only; @petewarden would probably know better about the other ones.

Reply:
I was going to just create an option tensorflow_BUILD_TOOLS in separate PR and build these tools conditionally.

Reply:
Right, but `proto_text` is needed *unconditionally* by the rest of the build process. As far as I know, it has no other uses, so it shouldn't be installed when we install TensorFlow.

Reply:
Removed proto_text install target.